node/light_client.go

```diff
@@ -0,0 +1,181 @@
+package node
+
+import (
+	"context"
+
+	"github.com/tendermint/tendermint/config"
+	tmbytes "github.com/tendermint/tendermint/libs/bytes"
+	rpcclient "github.com/tendermint/tendermint/rpc/client"
+	ctypes "github.com/tendermint/tendermint/rpc/core/types"
+	"github.com/tendermint/tendermint/types"
+)
+
+var _ rpcclient.Client = &LightClient{}
+
+type LightClient struct {
+	*types.EventBus
+	config *config.RPCConfig
```

Comment:
Please remove unused fields. We well add them if/when necessary.

Reply:
ok, but keeping the `EventBus` because it needs `IsRunning` to fit the interface